pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -325,7 +325,8 @@ func (rq *ResourceQuotaController) syncResourceQuota(v1ResourceQuota *v1.Resourc
 			Namespace:       resourceQuota.Namespace,
 			ResourceVersion: resourceQuota.ResourceVersion,
 			Labels:          resourceQuota.Labels,
-			Annotations:     resourceQuota.Annotations},
+			Annotations:     resourceQuota.Annotations,
```

Comment:
Are you looking for something like this: https://github.com/kubernetes/kubernetes/pull/45552